### PR TITLE
Typing Implementation

### PR DESCRIPTION
**Pull request by dkosteck**
Implemented typing hints for all common code (excluding test implementations).
Moved the testdata to configtestdata object.
Fixed unit test import error (needs review/potentially a better solution).
Fixed code style for flake8 Lint.
Made GitHub action runner for mocks run from root.

All tests report: https://gist.github.com/dkosteck/c75bba08fe2311e29c284dd8131c958f

### CODE REVIEW

File: sriov/common/test_utils_mock.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import unittest
 from mock import Mock, patch
+import sys, os
+sys.path.insert(1, os.path.dirname(os.path.abspath(__file__)))
```
Comment (jianzzha) [inline sriov/common/test_utils_mock.py:4]: @spenced any suggestion on this line in order to this script from a different folder?

**Comment by dkosteck** [comment]
Updated all test case report (QinQ failure expected): https://gist.github.com/dkosteck/757c21bed7611e2c50dac28a612bf82d

File: sriov/common/utils.py [hunk 2]
```diff
@@ -533,8 +547,8 @@ def execute_and_assert(ssh_obj, cmds, exit_code, timeout=0):
     return outs, errs
 
 
-def execute_until_timeout(ssh_obj, cmd, timeout=10):
-    """Execute cmd and check for 0 exit code until timeout
+def execute_until_timeout(ssh_obj: ShellHandler, cmd: str, timeout: int = 10) -> bool:
+    """ Execute cmd and check for 0 exit code until timeout
```
Comment (jianzzha) [inline sriov/common/utils.py:551]: @dkosteck @akiselev1 I thought the standard google style has no space here, and this space was removed earlier to pass the flake check, right?

Comment (jianzzha) [reply sriov/common/utils.py:551]: ACK. Just to be clear, here is an example of the google docstring,
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

**Comment by dkosteck** [comment]
All test: https://gist.github.com/dkosteck/460bcb74124a840a4aa2658c3d54f8e9
QinQ and Spoof off failing

**Comment by dkosteck** [comment]
All tests report: https://gist.github.com/dkosteck/45ef3b3102d95e3e98b677e512d26485

Now only QinQ if failing (as expected).

**Review by jianzzha — APPROVED** [review]
Thanks. Looks good to me!

**Pull request merged**